extension/content.js

```diff
@@ -393,7 +393,7 @@ document.addEventListener('DOMContentLoaded', () => {
 				showRealNames();
 			}
 
-			if (pageDetect.isCommit() || pageDetect.isSingleCommit()) {
+			if (pageDetect.hasDiff()) {
```

Comment:
This is is something which I wasn't completely sure about. See disclaimer in the commit message and #322. This is not working anyway, but when it is fixed, ideally it should use that check. Feedback is welcome.